python/dgl/distributed/partition.py

```diff
@@ -749,11 +724,11 @@ def get_homogeneous(g, balance_ntypes):
             node_parts = random_choice(num_parts, sim_g.number_of_nodes())
         start = time.time()
         parts, orig_nids, orig_eids = partition_graph_with_halo(sim_g, node_parts, num_hops,
-                                                                reshuffle=reshuffle)
+                                                                reshuffle=True)
         print('Splitting the graph into partitions takes {:.3f}s, peak mem: {:.3f} GB'.format(
             time.time() - start, get_peak_mem()))
         if return_mapping:
-            orig_nids, orig_eids = _get_orig_ids(g, sim_g, reshuffle, orig_nids, orig_eids)
+            orig_nids, orig_eids = _get_orig_ids(g, sim_g, True, orig_nids, orig_eids)
```

Comment:
why not remove the `reshuffle` argument in `_get_orig_ids()`? blocked by someone else uses this function?

Reply:
Line 690 set `reshuffle` to False, will remove this argument influence the normal behavior?

Reply:
oh, but leaves an argument named `reshuffle` looks a bit weird. we're supposed to remove all `reshuffle`. could you check the difference between `reshuffle=True` and `reshuffle=False` in this case? we need to try to remove it or we need add comments to clarify why we need this argument. what do you think

Reply:
removed for fix